src/main/java/com/googlecode/jsonrpc4j/IJsonRpcClient.java

```diff
@@ -36,7 +36,7 @@
 	 * Invokes the given method with the given argument.
 	 *
 	 * @param methodName the name of the method to invoke
-	 * @param arguments the arguments to the method
+	 * @param argument the arguments to the method
```

Comment:
You may want to change "the arguments to the method" to "the argument to the method" then